src/test/java/test/github1471/TestNgEnableBehaviour.java

```diff
@@ -0,0 +1,42 @@
+package test.github1471;
+
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import test.SimpleBaseTest;
+import test.enable.InvokedMethodListener;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Test test should ensure that disabled tests are not executed.
+ * See issue: https://github.com/cbeust/testng/issues/1471
+ *
+ * @author Kirusanth Poopalasingam ( code@kiru.io )
+ */
+public class TestNgEnableBehaviour extends SimpleBaseTest{
```

Comment:
This test class has no relation to the issue. Could you remove the commit from the branch?